quickstarts/video/video-web/video-web-akamai/config.yml

```diff
@@ -1,8 +1,28 @@
 id: 794055cb-eeb3-4cca-b478-6cd1fb97678e
 name: video-web-akamai
 title: Video agent for Akamai player
-description: Agent to monitor video applications using Akamai player.
-summary: Agent to monitor video applications using Akamai player.
+description: |-
+
+  ## Why monitor Akamai player?
+
+  Akamai Media Player is designed to help you insert quality media playback experiences into web, mobile, and connected-TV applications using SDKs. It includes SDKs for web, iOS, tvOS, Android, Android TV, Fire TV, and Chromecast. New Relic Akamai quickstart monitors your Akamai player with digital experience monitoring, tracing, diagnostics, and application analytics.
+  
+  ### Akamai player quickstart highlights
+  
+  The New Relic Akamai player quickstart has the following features
+  - Dashboards| Our dashboards proactively track metrics like video attempts, video plays, and average video start time. The dashboards also help you monitor other key indicators like connection buffering ratio, plays by app, and more.
+  - Alerts| You can get instant alerts on performance metrics like video playback failure, video start failure, and video start time.
+
+  ### How to monitor Akamai performance
+  
+  New Relic’s Akamai quickstart automatically instruments your Akamai player with our [video agent](https://developer.newrelic.com/instant-observability/video-web-akamai/794055cb-eeb3-4cca-b478-6cd1fb97678e/). It empowers you to monitor video applications with practical dashboards and alerts. The dashboard provides interactive visualizations to explore your data and understand context. In particular, the dashboard offers you the ability to drill down into performance details like the number of video attempts, number of video plays, and the overall average video start.
```

Comment:
Is this linking to itself? what's the reason to link the quickstart to it's own details page?

Reply:
@jpvajda no, that's incorrect. However, I don't love the links we do have. Let's remove the link for now and then I'll talk with Bradley about a better option.

Reply:
```suggestion
  New Relic’s Akamai quickstart automatically instruments your Akamai player with our video agent. It empowers you to monitor video applications with practical dashboards and alerts. The dashboard provides interactive visualizations to explore your data and understand context. In particular, the dashboard offers you the ability to drill down into performance details like the number of video attempts, number of video plays, and the overall average video start.
```